modules/internal/ChapelTuple.chpl

```diff
@@ -100,7 +100,7 @@ module ChapelTuple {
   pragma "no doc"
   proc *(param p: uint, type t) type {
     if p > max(int) then
-      compilerError("Tuples of size >" + max(int) + " are not currently supported");
+      compilerError("Tuples of size >" + max(int):string + " are not currently supported");
```

Comment:
Aside: we should probably lower this limit way down from max(int). We generate one field per each tuple component, so I have seen the compiler slow down significantly at 200k.

Reply:
I'm surprised we could get to 200k.  I thought it slowed down pretty badly even after double digits.

Reply:
It never passed resolution at 200k. So, putting a stop after double digits would be fine by me as well.